frontend/packages/component-library/src/accordion/accordion.module.scss

```
@@ -0,0 +1,75 @@
+@import '@code-dot-org/component-library-styles/colors.scss';
+@import '@code-dot-org/component-library-styles/mixins.scss';
+
+// Accordion common styles
+.accordion {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-items: center;
+  gap: 1rem;
+  width: 100%;
+
+  details {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+
+    border-radius: 0.25rem;
```

You can use the `$regular-border-radius` variable here:
```suggestion
    border-radius: $regular-border-radius;
```